wandb/sdk/launch/agent/agent.py

```diff
@@ -301,6 +302,16 @@ def run_job(self, job: Dict[str, Any]) -> None:
         _logger.info("Parsing launch spec")
         launch_spec = job["runSpec"]
 
+        # Abort if this job attempts to override the entrypoint and we're not
+        # configured to allow that.
+        if self._lock_entrypoint:
+            overrides = launch_spec.get("overrides", {})
+            if overrides.get("entry_point"):
+                raise ValueError(
```

Comment:
Won't this fully crash the agent?

Reply:
No because the agent calls this function in a blanket try/except

Reply:
https://github.com/wandb/wandb/blob/00a38a95f1f824679810f7e05e6ccd1e70d0d9a0/wandb/sdk/launch/agent/agent.py#L359